src/app.ts

```diff
@@ -118,7 +118,7 @@ const scanFiles = async (args: Args): Promise<string[]> => {
   const found = [];
 
   try {
-    const pattern = [mergedArgs.files, '!node_modules'];
+    const pattern = [mergedArgs.files, '!**/node_modules**', '!**/node_modules/**'];
```

Comment:
I think you can actually do this with this one glob:

```suggestion
    const pattern = [mergedArgs.files, '!**/node_modules'];
```

Reply:
Not really, this still matches more deeply nested files.

<img width="350" alt="Screen Shot 2020-10-18 at 5 29 32 PM" src="https://user-images.githubusercontent.com/2881382/96388672-a1df4600-1167-11eb-8e98-a109045e0aa5.png">

Reply:
I'm not sure which tool you're using to test these globs but I tried this code and it does exclude `node_modules` directories nested 0, 1 and 2 levels deep. I didn't have an example to test on with more nesting than that but it also didn't seem necessary.

Reply:
Huh, weird, I was using [this one](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%21%2A%2A%2Fnode_modules&matches=false&tests=%2F%2F%20This%20will%20match%20as%20it%20ends%20with%20%27.js%27&tests=%2Fhello%2Fworld.js&tests=%2F%2F%20This%20won%27t%20match%21&tests=%2Ftest%2Fsome%2Fglobs&tests=node_modules&tests=some_package%2Fnode_modules&tests=some_package%2Fnode_modules%2Fsome_deeper_package%2Fnode_modules&tests=node_modules%2FJSONStream%2Ftest%2Fkeys.js&tests=packages%2Fneo-adjudication%2Fnode_modules%2Faws-sdk%2Fapis%2Frekognition-2016-06-27.examples.json&tests=packages%2Fneo-audit-logger-client%2Fnode_modules%2Faws-sdk%2Fapis%2Frekognition-2016-06-27.examples.json&tests=packages%2Fneo-elasticsearch%2Fnode_modules%2Faws-sdk%2Fapis%2Frekognition-2016-06-27.examples.json&tests=packages%2Fneo-encryption%2Fnode_modules%2Faws-sdk%2Fapis%2Frekognition-2016-06-27.examples.json&tests=packages%2Fneo-queue%2Fnode_modules%2Faws-sdk%2Fapis%2Frekognition-2016-06-27.examples.json)

I thought that worked too, but got thrown off by it not excluding them in this tool.
I'll update with your suggestion then, thanks! 👍